docs/users/customization-extendibility.md

```diff
@@ -79,5 +79,14 @@ services:
       - solr:$DDEV_HOSTNAME
 ```
 
+## Providing custom nginx configuration
```

Comment:
Does it matter that the right name of this file would be customization-extensibility.md? I don't think so.